lib/search.rb

```diff
@@ -882,6 +882,15 @@ def tags_search
     end
   end
 
+  def exclude_topics_search
```

Comment:
Is this method called through some dynamic method lookup? Quick grep here makes it seem like the method isn't being used.